src/darwin/Framework/CHIP/MTRP256KeypairBridge.mm

```diff
@@ -82,7 +85,7 @@
         MTR_LOG_ERROR("ECDSA sign msg failure: no keypair to sign with.");
         return CHIP_ERROR_INCORRECT_STATE;
     }
-    NSData * msgData = [NSData dataWithBytes:msg length:msg_length];
+    NSData * msgData = AsData(chip::ByteSpan(msg, msg_length));
```

Comment:
This feels a bit unnecessary, creating a stack object of ByteSpan just to use AsData. I would leave this file as is.

Reply:
Done.